parser/config.go

```diff
@@ -176,9 +176,11 @@ Usage: deis config:push [options]
 
 Options:
   -a --app=<app>
-    the uniquely identifiable name for the application.
+    The uniquely identifiable name for the application.
   -p <path>, --path=<path>
-    a path leading to an environment file [default: .env]
```

Comment:
Sorry for giving you bad advice here about checking filepath first. I completely forgot about the `.env` default. It would be nice if we didn't have to go down this route, as it feels a little hacky. 

So much for the easy solution :smile:. I guess we do need to find the correct Sdin mode. If I have some time I'll help you experiment with this this week.

As for similar code in config:pull, [here](https://github.com/deis/workflow-cli/blob/master/cmd/config.go#L186) it is. It uses Stout, which means that redirecting a bash scripts output would likely run into similar issues to this.

Thanks so much for working on this and I apologize again for giving you bad advice :cry:!